members/SNJU36466_725072.yaml

```diff
@@ -1,7 +1,6 @@
-bioguide: SNJU923870df
+bioguide: SNJU36466_725072
```

Comment:
What happened here? Did the target change position or was an existing yaml reused/tweaked for a new target?

Reply:
It was an existing yaml reused for a new target